pages/index.tsx

```diff
@@ -1,65 +1,24 @@
-import React, { useState, useMemo } from "react";
 import Head from "next/head";
-import { AgGridReact } from "ag-grid-react";
-import data from "../utils/data";
-import "ag-grid-enterprise";
-import "ag-grid-community/dist/styles/ag-grid.css";
-import "ag-grid-community/dist/styles/ag-theme-alpine-dark.css";
 import Sidebar from "../components/Sidebar";
+import Table from "../components/Table";
+import NavbarItem from "../components/NavbarItem";
+import { ReactChild, ReactNode } from "react";
 
-export default function Home() {
-  //define the column header for our table in a local state
-  const [columnDefs, setColumnDefs] = useState([
-    { field: "country", rowGroup: true, enableRowGroup: true, hide: true },
-    { field: "sectors", rowGroup: true, enableRowGroup: true, hide: true },
-    {
-      headerName: "Activity Details",
-      children: [
-        { field: "title", enableRowGroup: true },
-        { field: "date", enableRowGroup: true },
-      ],
-    },
-    {
-      headerName: "Values",
-      children: [{ field: "signed amount" }],
-    },
-  ]);
-
-  //define the properties of our tables columns 
-  const defaultColDef = useMemo(() => {
-    return {
-      flex: 1,
-      minWidth: 100,
-      sortable: true,
-      resizable: true,
-    };
-  }, []);
-
-  //define the minimum width for a column group
-  const autoGroupColumnDef = useMemo(() => {
-    return {
-      minWidth: 200,
-    };
-  }, []);
-
+export default function Home():ReactChild {
   return (
     <div>
       <Head>
         <title>Ag grid with next js</title>
         <meta name="description" content="Ag grid created with next js" />
       </Head>
-      <div className="w-full h-full flex">
-        <Sidebar></Sidebar>
-        <div className="ag-theme-alpine-dark w-4/5 h-screen">
-          <AgGridReact
-            rowData={data}
-            columnDefs={columnDefs}
-            defaultColDef={defaultColDef}
-            autoGroupColumnDef={autoGroupColumnDef}
-            animateRows={true}
-            rowGroupPanelShow={"always"}
-          ></AgGridReact>
-        </div>
+      <div className="w-full h-full flex bg-[#151718] gap-1 p-1 font-primary">
+        <Sidebar>
+          <NavbarItem to={"/"}>country</NavbarItem>
+          <NavbarItem to={"/"} cls="opacity-60">sectors</NavbarItem>
+          <NavbarItem to={"/"} cls="opacity-60">signed amount</NavbarItem>
+          <NavbarItem to={"/"} cls="opacity-60">date</NavbarItem>
```

Comment:
move these links to an array, and handle the active link style dynamically from inside `NavbarItem` component